src/Stripe.net/Services/StripeBaseOptions.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections.Generic;
+
+namespace Stripe
+{
+    public class StripeBaseOptions
+    {
+        public void AddExtraParam(string key, string value) {
+            ExtraParams.Add(key, value);
+        }
+
+        public Dictionary<string, string> ExtraParams = new Dictionary<string, string>();
```

Comment:
@ob-stripe did we want to figure out how to make that one not public before merging?